app/frontend/src/constants.ts

```
@@ -0,0 +1,83 @@
+export const CHAT_STORE: IndexedDBStorage = {
+    db_name: "MUCGPT-CHAT",
+    objectStore_name: "chat",
+    db_version: 2
+};
+
+// Local storage
+export const enum STORAGE_KEYS_CHAT {
+    CHAT_TEMPERATURE = "CHAT_TEMPERATURE",
+    CHAT_SYSTEM_PROMPT = "CHAT_SYSTEM_PROMPT",
+    CHAT_MAX_TOKENS = "CHAT_MAX_TOKENS"
+}
```

Biome (1.9.4)

[error] The enum declaration should not be const

Const enums are not supported by bundlers and are incompatible with the 'isolatedModules' mode. Their use can lead to import inexistent values.
See TypeScript Docs for more details.
Safe fix: Turn the const enum into a regular enum.


(lint/suspicious/noConstEnum)